source/Item.hx

```diff
@@ -0,0 +1,86 @@
+package;
+import flixel.FlxG;
+import flixel.FlxObject;
+import flixel.FlxSprite;
+import flixel.system.FlxAssets.FlxGraphicAsset;
+import haxe.Timer;
+
+
+/**
+ * ...
+ * @author Cameron Yuan
+ */
+class Item extends FlxSprite
+{
+	private var player_trace:Player;
```

Comment:
On a side note, implementation could be changed to keep track of an FlxObject and keep the tracker within the class. This would allow multiplayer item memory as well as allowing enemies to have specific Items. I will do as you suggested, but this is a potential implementation